libs/cardano-ledger-test/cardano-ledger-test.cabal

```diff
@@ -79,6 +79,7 @@ library
     cardano-ledger-alonzo,
     cardano-ledger-alonzo-test,
     cardano-ledger-babbage,
+    cardano-ledger-binary:testlib,
```

Comment:
Could you please extract these property tests changes added to `cardano-ledger-test` into a separate PR. I would prefer not to mix implementation from `cardano-ledger-binary` and `cardano-binary` at this point yet.

Reply:
I moved the tests to #3095

Reply:
Cool. Thank you.

Reply:
@Soupstraw I was actually talking about the changes to `cardano-ledger-test` and `cardano-data` packages. Everything else was good stuff, you didn't really need to revert it